### ISSUE TITLE
[a11y] color combinations break fail AA/ AAA requirements

### ISSUE THREAD

**Issue by lislis** [okfde/jh-lauch-theme#47]
Problem:
- weiße Schrift auf $softblue bg
- weiße Schrft auf $softgreen bg
